features/journalist_can_create_article.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Journalist can create articles
+
+    As a journalist
+    In order write articles
+    I would like to be able to create an article
+
+    Scenario: Journalist can create article
+        Given we have 1 user with email "jocke@craft.se" and role "journalist"
```

Comment:
Put this in the background and create a table for it.

Reply:
done